crystal-map-processor/src/main/java/com/schwarz/crystalprocessor/model/id/DocIdHolder.kt

```diff
@@ -23,7 +23,7 @@ class DocIdHolder(docId: DocId, val customSegmentSource: MutableList<DocIdSegmen
         }
     }
 
-    val pattern = docId.value
+    val pattern = docId.value.replace(":null:", "::").replace(":null",":")
```

Comment:
ah that was your comment, right? `.replace("null:", ":")` would only cover the case of an empty prefix, that is why you did not add it'

Reply:
Yes. I think an empty prefix is a broken DocId.